monetio/models/raqms.py

```diff
@@ -52,10 +74,27 @@ def open_mfdataset(fname, *, surf_only=False):
             "in netCDF format."
             "Do not mix and match file types."
         )
-
-    ds = xr.open_mfdataset(names, concat_dim="time", drop_variables=["theta"], combine="nested")
+    if var_list is not None:
+        var_list.extend(['lat','lon',"IDATE", "Times","psfc","delp",'pdash','ttheta'])
+        ds = xr.open_mfdataset(names, concat_dim="time", drop_variables=["theta"], combine="nested")[var_list]
+    else:
+        ds = xr.open_mfdataset(names, concat_dim="time", drop_variables=["theta"], combine="nested")
```

Comment:
Most of these two lines are the same, so you could move it above the `if` and just do `ds = ds[var_list]` inside the `if`.